### DISCUSSION TITLE
Is gravity being induced via plasma configurations?

### DISCUSSION THREAD

**Discussion by gfellow** [a-cosmology-group/acg#190] [Gravitation Models]
Nature is universally efficient and eloquent. It makes no sense that something a precious as matter would be poured into the dumpsters that we presently refer to as black holes.
Having been fascinated and studied plasma configurations for over forty years, it seems there is a strong possibility that these are capable of inducing gravity - gravity without a corresponding quantity of mass. 
Furthermore, I believe this as yet not realized phenomenon can be verified through laboratory experimentation, and if verified, would have no small impact on how we understand the Universe, a Universe in which plasma is on steroids:
"[A proposed induced gravity laboratory experiment](https://youtu.be/PNzaLRozQD4?si=NFLJ-SgYr4Bsr6Qo)"

**Comment by RedshiftDrift** [comment]
Stephen, the video has a fair numbers of inaccuracies:
1) There are no lasers that can detect a change in position of the beam to an accuracy of 1E-15 cm.  What stable lasers do is to measure length variations of an interferometer to 1E-15 cm.  So the experiment cannot work as proposed.
2) These ultra-stable lasers operate under fluorescent lights (plasma tube) in the laboratory.  The plasma in these lights does not have any effect on the lasers two meters below.  The claim that this experiment has not been done is false.  I have done this experiment and no anomaly has been reported: the experiment does not detect the plasma in the lights.
3) Thunder is produced by thermal expansion of air from the energy released in the discharge, not a "gravitational collapse".  When lightning goes through a tree, the tree explodes - it does not "collapse under induced gravity".
4) If a discharge generated enough gravity to bend light, a fluorescent light would "induce gravitational" attraction that would be noticeable with simple material structure.  But a fluorescent light does not collapse a house of cards or an unstable Jenga tower.
5) The video does not provide any quantitative estimate of how strong the "induced gravity" should be.  This is therefore compatible with 'zero'.

**Reply by gfellow** [reply]
Redshiftdrift, thank you for your comments.

_The claim that this experiment has not been done is false. I have done this experiment and no anomaly has been reported: the experiment does not detect the plasma in the lights._

With respect, your means of verification is to weak to yield a result. 

_Thunder is produced by thermal expansion of air from the energy released in the discharge, not a "gravitational collapse". When lightning goes through a tree, the tree explodes - it does not "collapse under induced gravity"._

Observations of explosive experiments, in the the first initial fraction of a millisecond, the event is an implosion. The violent outward force is a secondary event. The visual effect of a tree blasted apart that you give as an example is included in that category. 
Wikipedia: "cavitation voids involve the local deposition of energy, such as an intense focused laser pulse (optic cavitation) or with an electrical discharge through a spark."

_If a discharge generated enough gravity to bend light, a fluorescent light would "induce gravitational" attraction that would be noticeable with simple material structure._ 

I would suggest that using a florescent tube to determine such a measurement is too simplistic. One cannot get any result, as the photon discharge interaction is way too weak to determine by the method suggested.
The polarized photons have to pass extremely close to the discharge, and their trajectory has to be amplified by extending the travel distance that the photon propagates after passing the discharge.

_The video does not provide any quantitative estimate of how strong the "induced gravity" should be. This is therefore compatible with 'zero'._

The only measure presently available will be the deviation of a photon. One would look for a 'yes or a 'no'.
It would be premature to be expect estimates of relative strengths pertaining to the momentary induced gravitational field. The photon either deviates or it does not.

**Reply by bgreenman** [reply]
The proton deviation is the only measurement we have to detemine if gravity can collapse in a plasma. that might not not be enough to prove it.

**Reply by gfellow** [reply]
Did you mean the 'photon'?

**Reply by bgreenman** [reply]
yes! I mistyped

**Comment by gfellow** [comment]
Says who?😄
-S-

**Comment by anshandeecho-cpu** [comment]
Sorry, English is indeed a major challenge for me when it comes to reading.
I need translation tools to understand the specific content. If you're
interested, I can send you my treatise tomorrow—I specifically wrote a
section on how force is generated by frequency pulling.

gfellow ***@***.***> 于 2026年3月28日周六 19:51写道：

**Comment by gfellow** [comment]
Sure! Send it along, I'd love to read it.
Best, Stephen
